libyul/AsmAnalysis.cpp

```diff
@@ -339,10 +345,10 @@ vector<YulString> AsmAnalyzer::operator()(FunctionCall const& _funCall)
 		m_errorReporter.typeError(
 			7000_error,
 			_funCall.functionName.location,
-			"Function \"" + _funCall.functionName.name.str() + "\" expects " +
-			to_string(parameterTypes->size()) +
-			" arguments but got " +
-			to_string(_funCall.arguments.size()) + "."
+			"Function {:q} expects {} arguments but got {}.",
```

Comment:
Tell me, quick, what is the name of the function, what is the expected number of arguments and how many were actually supplied?